### ISSUE TITLE
/assets/js/erpnext_pk.min.js missing

### ISSUE THREAD

**Issue by nbhatti** [ParaLogicTech/erpnext_pk#13]
Code refers to /assets/js/erpnext_pk.min.js but this file seems missing. Can you share?

**Comment by SaiFi0102** [comment]
It's auto generated by the `bench build` command... This app is currently supported on version-13 only

**Comment by nbhatti** [comment]
Yes I figured that out, v13 does build that but in v14 it's not building the assets properly.

**Comment by ggraza** [comment]
How to resolve 
GET http://erp.domainname.com/assets/js/erpnext_pk.min.js net::ERR_ABORTED 404 (Not Found)
on v14
